src/main/java/io/github/syst3ms/skriptparser/expressions/ExprElement.java

```diff
@@ -0,0 +1,138 @@
+package io.github.syst3ms.skriptparser.expressions;
+
+import io.github.syst3ms.skriptparser.Main;
+import io.github.syst3ms.skriptparser.lang.Expression;
+import io.github.syst3ms.skriptparser.lang.TriggerContext;
+import io.github.syst3ms.skriptparser.parsing.ParseContext;
+import io.github.syst3ms.skriptparser.util.math.NumberMath;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.concurrent.ThreadLocalRandom;
+
+/**
+ * A certain element or multiple elements out of a list of objects.
+ * Remember that in Skript, indices start from 1.
+ *
+ * @name Element
+ * @type EXPRESSION
+ * @pattern ([the] first|[the] last|[a] random|[the] %*integer%(st|nd|rd|th)) element out [of] %objects%
+ * @pattern [the] (first|last) %*integer% elements out [of] %objects%
+ * @pattern %objects%\[%*integer%\]
+ * @since ALPHA
+ * @author Mwexim
+ */
+public class ExprElement implements Expression<Object> {
+
+	static {
+		Main.getMainRegistration().addExpression(
+				ExprElement.class,
+				Object.class,
+				false,
+				"(0:[the] first|1:[the] last|2:[a] random|3:[the] %*integer%(st|nd|rd|th)) element out [of] %objects%",
+				"[the] (0:first|1:last) %*integer% elements out [of] %objects%",
+				"%objects%\\[%*integer%\\]");
```

Comment:
The indices shouldn't be literals: it can surely prove useful to plug in something like a loop-value in there.